src/main/host/host.rs

```diff
@@ -716,6 +717,29 @@ impl Host {
         };
     }
 
+    /// Returns the specified process.
+    pub fn get_process(
```

Comment:
Nit - I  think it's more conventional in Rust not to have the `get_` prefix.

Reply:
Actually it looks the current caller could use `process_borrow` instead?

Reply:
I missed that one. Have removed `get_process`.